app/Controller/PagesController.php

```diff
@@ -74,3 +120,44 @@ public function display() {
 		}
 	}
 }
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
+class Foo {
+    public function example()  {
+        if ($a == $b)  {
```

Comment:
__[phpmd]__
* Avoid variables with short names like $a. Configured minimum length is 3.
* Avoid variables with short names like $b. Configured minimum length is 3.
* Avoid unused local variables such as '$a'.
* Avoid unused local variables such as '$b'.